app/policies/application_policy.rb

```diff
@@ -0,0 +1,49 @@
+class ApplicationPolicy
```

Comment:
Do u need this?

Reply:
The documentation said that the all policy's classes should inherit from that class, but i don't know exactly